test/lib/circular_buffer_uint/constructor.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const Utils = require('../../test_lib/utils.js');
+const BN = require('bn.js');
+
+const CircularBufferUint = artifacts.require('CircularBufferUint');
+
+contract('CircularBufferUint.constructor()', async (accounts) => {
```

Comment:
There should be a test for the positive scenario.

Reply:
➕ added